assets/js/blocks/cart-checkout/cart/full-cart/index.js

```diff
@@ -104,14 +103,8 @@ const Cart = ( {
 		removeCoupon,
 		isApplyingCoupon,
 		isRemovingCoupon,
-		cartCouponsErrors,
```

Comment:
I assume that these props are still being passed, and that we'll do something with them in follow up?

Reply:
They are still available via the hook. They weren't "passed" anywhere here, just used to trigger throwing an error.

Of note is #1953 is where I'm going to implement similar notice handling for other types of errors rather than throwing an error that gets caught by the error boundary.